google/resource_google_project.go

```diff
@@ -346,6 +337,18 @@ func resourceGoogleProjectUpdate(d *schema.ResourceData, meta interface{}) error
 	return resourceGoogleProjectRead(d, meta)
 }
 
+func updateProjectOverwriteOnSuccess(config *Config, d *schema.ResourceData, project_name string, projectPtrPtr **cloudresourcemanager.Project) error {
```

Comment:
`**Project` feels a little awkward, we generally don't use double-pointers in TPG. I think that we can just pass in a `*Project`, and return `newProj` as a `*Project` on success, replacing `p` at the callsite.

```suggestion
func updateProjectOverwriteOnSuccess(config *Config, d *schema.ResourceData, projectName string, desiredProject *cloudresourcemanager.Project) (*cloudresourcemanager.Project, error) {
```

Reply:
That sounds fine, will change.